src/templating/actions.ts

```diff
@@ -10,9 +11,9 @@ export async function downloadTemplate(
   const files = await getTemplateFiles(templateName);
   try {
     await writeFiles(files, targetDirectory, bundleName);
-    console.log(chalk`{green SUCCESS} Created new bundle ${bundleName}`);
+    logger.info(chalk`{green SUCCESS} Created new bundle ${bundleName}`);
```

Comment:
Should this use `writePlainOutput` instead of logging as it is a specific output to the user?